### PR TITLE
Add a link to the Flutter Q2 survey

### PR DESCRIPTION
**Pull request by DanTup**
@pq

### CODE REVIEW

**Comment by DanTup** [comment]
(don't worry about the failed builds, I landed some tests that will fail until a few components release over the coming days, they'll be green before this merges :-))

File: src/user_prompts.ts [hunk 1]
```diff
@@ -41,7 +41,77 @@ export function showUserPrompts(context: Context, workspaceContext: WorkspaceCon
...
+
+	// Don't show this notification if user previously said not to.
+	if (doNotShow)
+		return false;
+
+	// Don't show this notification if we've shown it in the last 40 hours.
+	if (lastShown && now - lastShown < fortyHoursInMs)
+		return false;
+
+
```
Comment (pq) [inline src/user_prompts.ts]: I think this extra space is problematic for `tslint`?

Comment (DanTup) [reply src/user_prompts.ts]: Ha, good spot! One day I'll figure out why tslint issues seem to be unreliable showing in the editor... I'll fix this tomorrow when the other stuff is done, so it should all go green. Thanks!

**Comment by DanTup** [comment]
Builds all green except a flake (timeout) in the AppVeyor master-merge one, so merging this.

**Pull request merged**